test/e2e/multihoming.go

```diff
@@ -14,10 +14,12 @@ import (
 	. "github.com/onsi/gomega"
 
 	"github.com/docker/docker/client"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
this dependency is already present in the line below ...

I agree the original name was unfortunate, and I'd rather have this one, but, IMHO, we should stick to using `v1` now, and later on refactor by renaming the import.